jaxrs-api/src/main/java/jakarta/ws/rs/client/ClientResponseContext.java

```diff
@@ -88,6 +89,19 @@ public interface ClientResponseContext {
      */
     public String getHeaderString(String name);
 
+    /**
+     * Checks whether a header with a specific name and value (or item of the comma-separated value list) exists.
+     *
+     * @param name the message header.
+     * @param valuePredicate value must fulfil this predicate.
+     * @return {@code true} if and only if a header with the provided name exists having either the exact value or whose
+     * comma-separated header string contains value as a whole word.
```

Comment:
The phrase "whole word" seems rather imprecise. Could we describe this better in terms of spaces etc? Perhaps add an example in the doc too.

Reply:
I leftover from the original proposal. You are right, this should get adapted to the latest method signature. What about `...comma-separated header string contains one ore more separated values each matching the predicate.`? A bit clumsy but I rather precide.

Reply:
I changed the JavaDocs. Hope it now is unambiguous.

Reply:
@spericas Agreed?